server/liveblog/blogs/tasks.py

```diff
@@ -54,7 +55,33 @@ def publish_embed(blog_id, theme=None, output=None, api_host=None):
             f"Failed embed generation with theme `{theme}`. Error: {e}",
             exc_info=exc_info,
         )
-        return notify_about_embed_generation_error(str(e), blog_id, theme)
+        notify_about_embed_generation_error(str(e), blog_id, theme)
+
+        logger.info('generate_embed_fallback for blog "{}" started.'.format(blog_id))
+        try:
+            theme = "default"
```

Comment:
If for some reason the theme failing is "default" theme, we would end up in an infinite loop of embed generation tasks. So we need to skip this if the theme is already default plus avoid telling the client that it will fallback to Default.

Reply:
also, maybe we could extract this into separate function like `def fallback_to_default_theme_if_needed(self, ...` or something similar.